api/middleware/authorization.ts

```diff
@@ -1,15 +1,18 @@
-import jwt from 'jsonwebtoken';
-import { Request, Response, NextFunction } from 'express';
+import { Response, NextFunction } from 'express';
+import { UserInfoRequest } from './requests';
 
-module.exports = async (request: Request, response: Response, next: NextFunction) => {
+const jwt = require('jsonwebtoken');
+
+module.exports = async (request: UserInfoRequest, response: Response, next: NextFunction) => {
   try {
     const jwtToken = request.header('token');
 
     if (!jwtToken) {
       return response.status(403).json('Not authorize');
     }
 
-    const payload = jwt.verify(jwtToken, process.env.SECRET);
+    const secret: string = process.env.SECRET || '';

```

Comment:
Dlaczego  || ''? Czy nie tracimy przez to zabezpieczeń?

Reply:
Jedyne co by się stało to brak weryfikacji gdy nie byłoby klucza w envie.

Reply:
Ok